.coderabbit.yml

```
@@ -26,6 +26,10 @@ reviews:
       This project uses tabs instead of spaces, do not suggest usage of spaces over tabs.
       Are there any 'red flags' in this code that might warrant closer investigation from a security standpoint?
       Explain what makes them suspicious.
+      When providing in line code suggestions, always use the following format:
+      ```suggestion
+      <code changes>
```

I'm not sure this is going to always be appropriate. I think suggestions are the default for reviews when it is confident, and you can always ask in comment replies

---

As you have seen, it is pretty hard to convince code rabbit of the proper form when asking for suggestions:

```suggestion
      When someone asks for code suggestions in an inline comment, always use the following format:
      ```suggestion
      <code changes>
```

---

As you have seen, it is pretty hard to convince code rabbit of the proper form when asking for suggestions:

```suggestion
      When providing code suggestions, particularly when requested, ensure GitHub's suggestion format is used, i.e.:
      ```suggestion
      <code changes>
```